src/components/organisms/settings/organization/billing/organizationBilling.tsx

```diff
@@ -42,7 +68,11 @@ export const OrganizationBilling = () => {
 		if (!usage) return null;
 		return usage.usage.find((item) => item.limit === limitName);
 	};
-	const projectsUsage = getUsageForLimit("projects");
+	const usageItems = [

```

Comment:
This always calls all three `get*` functions, is that intended?

Reply:
Hey, it calls only this function, which should only extract the relevant key from the object:
```
const getUsageForLimit = (limitName: string) => {
		if (!usage) return null;
		return usage.usage.find((item) => item.limit === limitName);
	};
```

When the user visit this page, we fetch the usage data using `getUsage();` and in this code we only divide it by the resources (projects/events/sessions)